### ISSUE TITLE
Pico W internal light

### ISSUE THREAD

**Issue by zackify** [microsoft/devicescript#402]
I don't see a way to turn on the internal light on the pico w with the startLightBulb command. I searched the issues a lot and the docs. Sorry if this was explained elsewhere.

**Comment by pelikhan** [comment]
You are right, looks like we haven't updated the hardware configuration with led info 

https://microsoft.github.io/devicescript/devices/rp2040/pico-w

**Comment by pelikhan** [comment]
It looks like the LED is connected to the wireless chip, not the RP2040, on the pico-w so we cannot control it with the startLightbulb function.

**Comment by pelikhan** [comment]
@mmoskal can we update the board description to callout that the LED is not available?
